src/Utilities/api.js

```diff
@@ -44,4 +44,11 @@ export const postComment = (article_id, newCommentText, user) => {
 
 export const deleteComment = (comment_id) => {
     return ncNews.delete(`/comments/${comment_id}`)
-}
\ No newline at end of file
+}
+
+export const getArticlesByTopic = (topic) => {
+    return ncNews.get(`articles?topic=${topic}`)
+    .then((response) => {
+        return response.data.articles
+    })
```

Comment:
Instead of creating a new api call, you can reuse your existing getArticles function. Have a look at axios queries. You can give a get request an object containing a query. If it's there it will use it, if not it will return all of the articles. 

If you implement this, you might be able to clean up your useEffect too!